foundation/foundation-mda/src/main/resources/templates/deployment/versioning/versioning.values-dev.yaml.vm

```diff
@@ -1,4 +1,4 @@
 # Dev values for versioning.
-aissemble-fastapi-chart:
-  image:
-    imagePullPolicy: IfNotPresent
\ No newline at end of file
+image:
```

Comment:
Q: This is being reverted because this values file template is only used for the v1 generation, and not v2?

Reply:
Yes, this was a bug identified in testing